presentation/PresentationConnection_basic-manual.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2016 Intel Corporation.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of works must retain the original copyright notice, this list
+  of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the original copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+* Neither the name of Intel Corporation nor the names of its contributors
+  may be used to endorse or promote products derived from this work without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<meta charset="utf-8">
+<title>PresentationConnection basic attributes for Controlling User Agent</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="author" title="He Yue" href="mailto:yue.he@intel.com">
+<link rel="help" href="https://w3c.github.io/presentation-api/#interface-presentationconnection">
+<div id="log"></div>
+<div class="content">
+  <h2>Test Steps:</h2>
+  <ol>
+    <li>Click the "start" button</li>
+    <li>Click the "close" button</li>
+  </ol>
+  <h2>Expected Result:</h2>
+  <ol>
+    <li>Info "start success" was displayed and no Error displayed</li>
+    <li>Info "close success" was displayed</li>
+  </ol>
+  <div align="middle">
+    <input type='button' value='start' onclick="onclickStart()" />
+  </div>
+  <div align="middle">
+    <input type='button' value='close' onclick="onclickClose()" />
+  </div>
+  <div id="result" style="width:100%; height:100px; word-wrap: break-word; word-break: normal;"></div>
+</div>
+<script type='text/javascript'>
+  var connection = null;
+  var url = "display/presentation.html";
+  function onclickStart() {
+    var request = new PresentationRequest(url);
+    request.start().then(function(conn) {
+      connection = conn;
+      document.getElementById("result").innerHTML = "start success" + "<br/>"; 
+      if(conn instanceof PresentationConnection) {
```

Comment:
As under testharness, the presentation API start() will encounter “requires user gesture” error (Louay figured out this problem in the meeting W3C Second Screen F2F).
So these test cases are defined as manual.